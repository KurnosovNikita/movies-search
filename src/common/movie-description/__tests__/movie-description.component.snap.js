import React from 'react';
import {shallow} from 'enzyme';
import MovieDescription from '../movie-description.component';

describe('MovieDescription component', () => {
    let props;
    beforeEach(() => {
        props = {
            selectedMovie: [{
                title: 'Harry Potter',
                runtime: '250',
                tagline: 'Film tagline',
                overview: 'This film is about greatness',
                release_date: '20.05.2018',
                poster_path: 'https://puu.sh/Dl8Lx/d4fcc4e5e6.png',
            }],
            changeToGlobalSearchPage: jest.fn(),
        };
    });

    it('should render <MovieDescription> correctly', () => {
        const component = shallow(<MovieDescription {...props}/>);

        expect(component).toMatchSnapshot();
    });
});
