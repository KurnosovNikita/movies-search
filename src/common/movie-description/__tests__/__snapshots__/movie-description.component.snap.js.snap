// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDescription component should render <MovieDescription> correctly 1`] = `
<header
  className="header header-movie-detail"
>
  <div
    className="header-context"
  >
    <Text
      className="title"
      text="netflixroulette"
    />
    <Button
      className="back-to-search-btn"
      id="backToSearch"
      onClick={[MockFunction]}
      text="search"
    />
    <div
      className="description-container"
    >
      <Image
        className="image"
        src="https://puu.sh/Dl8Lx/d4fcc4e5e6.png"
      />
      <div
        className="description"
      >
        <Text
          className="title"
          text="Harry Potter"
        />
        <Text
          className="subtitle"
          text="Film tagline"
        />
        <div
          className="sub-description"
        >
          <Text
            className="year"
            text="NaN"
          />
          <Text
            className="duration"
            text="250 min"
          />
        </div>
        <Text
          className="overview"
          text="This film is about greatness"
        />
      </div>
    </div>
  </div>
</header>
`;
